week-1-ts-ecosystem/w01e02-enter-the-world-of-typescript/src/type.ts

```diff
@@ -0,0 +1,27 @@
+
+export type ReturnType = HTMLDivElement | HTMLElement | HTMLParagraphElement
+
+export  type CartItem = {
```

Comment:
To jest typ będący jednocześnie pokazaniem "modelu danych"
jaki trzymamy w aplikacji.

`CartItem` w mojej opinii zasługuje na osobny plik (a nawet katalog `model`)
tak abyśmy wykorzystali typescript do pokazania tego jakiego rodzaju dane "biznesowe" przechowujemy.